man/systemd.exec.xml

```diff
@@ -1438,7 +1438,13 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         executed by the unit processes except for the listed ones will result in immediate process termination with the
         <constant>SIGSYS</constant> signal (whitelisting). If the first character of the list is <literal>~</literal>,
         the effect is inverted: only the listed system calls will result in immediate process termination
-        (blacklisting). If running in user mode, or in system mode, but without the <constant>CAP_SYS_ADMIN</constant>
+        (blacklisting). If the setting is prefixed with <literal>~</literal>, each system call or set of system call
```

Comment:
"Takes precedence of" doesn't sound right. Also, I don't think it's necessary to repeat that this is not allowed for whitelisted calls. Let's just say that it's only allowed for blacklisted ones. What about the following text instead:

`Blacklisted system calls and system call groups may optionally be suffixed with a colon (<literal>:</literal>) and <varname>errno</varname> error name such as <constant>EPERM</constant>, <constant>EACCES</constant> or <constant>EUCLEAN</constant>.
This value will be returned when a blacklisted system call is triggered, instead of terminating the processes immediately. This value takes precedence over the one given in  <varname>SystemCallErrorNumber=</varname>. If running in user mode, ...`

Reply:
Thanks.